easybuild/easyconfigs/p/PETSc/PETSc-3.12.3-foss-2019b.eb

```diff
@@ -19,6 +19,8 @@ checksums = [
     '64cf9d5008d5e92117e65bdec5316d991b6a6b8c8ecf7ea46eb790a498266297',  # PETSc_ranlib-fix.patch
 ]
 
+osdependencies = [('numpy', 'python-numpy')]
```

Comment:
I'm really not in favor of this...

The proper way forward is to replace this `PETSc` easyconfig with a new one that depend on `Python` (and has the corresponding `versionsuffix`), and fixing any easyconfigs that depend on it as well.

Not doing so was a mistake, so we should fix it properly, not sweep it under the rug...

Reply:
My intention was to ensure that current installations of `PETSc` in production can be reproduced in the future. Completely replacing the existing easyconfigs is also an option of course. In such a case we can rely on older EB releases to reproduce current installations.

Reply:
Indeed, I think that's the best way forward...

Sometimes bug fixes warrant "breaking" changes, this is a case like that imho...